qp_shogun/sortmerna/databases/rRNA_databases/scripts/generate_databases.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
The fasta files are pretty big, perhaps worth deleting them and adding their gz version?

Reply:
Compressed files.